### PR TITLE
Merge `8.1.x` into `develop`

### PR DESCRIPTION
**Pull request by nicosomb**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Merge `8.1.x` into `develop`
| Type?             | improvement 
| Category?         | ME 
| BC breaks?        | no
| Deprecations?     | no
| How to test?      | CI 🟢 
| Sponsor company   | PrestaShop SA

### CODE REVIEW

**Comment by nicosomb** [comment]
Automated tests: https://github.com/nicosomb/ga.tests.ui.pr/actions/runs/6009941742

**Review by Progi1984 — APPROVED** [review]
Ok for UI tests (@boubkerbribri)

File: admin-dev/themes/new-theme/js/components/grid/extension/position-extension.ts [hunk 1]
```diff
@@ -85,6 +85,9 @@ export default class PositionExtension {
             .removeClass('hover');
         },
       );
+
+    this.setReorderButtonLabel();
+    this.getReorderButton().on('click', (event) => this.oncClickReorderButton(event));
```
Comment (matks) [inline admin-dev/themes/new-theme/js/components/grid/extension/position-extension.ts:90]: Not related to this PR but I can't help notice this `oncClick` (extra "c")

**Review by matks — APPROVED** [review]

File: admin-dev/themes/new-theme/js/components/grid/extension/position-extension.ts (hunk 1 above)
Comment (boherm) [reply admin-dev/themes/new-theme/js/components/grid/extension/position-extension.ts:90]: Ow.. Sorry!? I think that is my fault 😅

**Review by mflasquin — APPROVED** [review]

**Review by M0rgan01 — APPROVED** [review]

**Review by boherm — APPROVED** [review]

File: admin-dev/themes/new-theme/js/components/grid/extension/position-extension.ts (hunk 1 above)
Comment (matks) [reply admin-dev/themes/new-theme/js/components/grid/extension/position-extension.ts:90]: Yes https://github.com/PrestaShop/PrestaShop/commit/3711c8656ddf9f8b9e18d2091a6655941a7e3ede#diff-b1915840145517da2bfae76b45fdb6fd83a8f42a1f49c9a9e538d99677d82f7aR87

Little PR to fix that? 😉

**Review by boubkerbribri — APPROVED** [review]

**Comment by boubkerbribri** [comment]
Thanks @nicosomb

**Pull request merged**